services/autorust/codegen/src/cargo_toml.rs

```diff
@@ -28,9 +33,12 @@ pub fn create(crate_name: &str, tags: &[&Tag], default_tag: Option<&str>, path:
 [package]
 name = "{}"
 version = "0.2.0"
-edition = "2018"
+edition = "2021"
 license = "MIT"
 description = "generated REST API bindings"
+repository = "https://github.com/azure/azure-sdk-for-rust"
+homepage = "https://github.com/azure/azure-sdk-for-rust"
+documentation = "https://docs.rs/{}"
```

Comment:
Since we're using `format!()` could we switch to named parameters? As more replacements are added it gets more difficult to keep them properly serialized.

Reply:
I'd like to switch over to a `askama` "jinja" template in a future pull request for this too.